src/category_theory/monoidal/functor.lean

```diff
@@ -0,0 +1,180 @@
+-- Copyright (c) 2018 Michael Jendrusch. All rights reserved.
+-- Released under Apache 2.0 license as described in the file LICENSE.
+-- Authors: Michael Jendrusch, Scott Morrison
+import category_theory.monoidal.category
+
+open category_theory
+open tactic
+
+universes v₁ v₂ v₃ u₁ u₂ u₃
+
+open category_theory.category
+open category_theory.functor
+
+namespace category_theory.monoidal
```

Comment:
I don't think we need this namespace. I think that `monoidal_category`, `lax_monoidal_functor`, and `monoidal_functor` should suffice as namespaces.

Reply:
Removed.